### PR TITLE
Removal of the historical .txt trace file format and related scripts

### LINKED ISSUE #760: Bug in OSI Docu on Tracefiles
#### Describe the bug
The page in our documentation on OSI trace files seems broken.
There seems to be a duplicate in heading and text. @philipwindecker is the documentation build pipeline the problem here?

Here the page in our docu: https://opensimulationinterface.github.io/osi-antora-generator/asamosi/latest/interface/architecture/trace_file_formats.html
Here the source for that page: https://github.com/OpenSimulationInterface/open-simulation-interface/blob/master/doc/architecture/trace_file_formats.adoc

### PR DESCRIPTION
**Pull request by PhRosenberger**
﻿#### Reference to a related issue in the repository
This PR fixes #760 

#### Add a description
Add a description of the changes proposed in the pull request.

**Some questions to ask**:
The historical .txt output format for hybrid binary/human readable OSI trace files does not really help anyone and was not used by the community so far. I therefore strongly propose to get rid of it and the related python script for conversion to the actually binary .osi format.

#### Take this checklist as orientation for yourself, if this PR is ready for the Change Control Board:
- [x] My suggestion follows the [style and contributors guidelines](https://opensimulationinterface.github.io/osi-antora-generator/asamosi/latest/specification/contributing/start_contributing.html).
- [x] I have taken care about the [message documentation](https://opensimulationinterface.github.io/osi-antora-generator/asamosi/latest/specification/contributing/commenting_messages.html) and the [fields and enums documentation](https://opensimulationinterface.github.io/osi-antora-generator/asamosi/latest/specification/contributing/commenting_fields_enums.html).
- [x] I have done the [DCO signoff](https://opensimulationinterface.github.io/osi-documentation/open-simulation-interface/doc/howtocontribute.html#developer-certification-of-origin-dco).
- [x] My changes generate no errors when passing CI tests. 
- [x] I have successfully implemented and tested my fix/feature locally.
- [x] Appropriate reviewer(s) are assigned.

### CODE REVIEW

**Review by ClemensLinnhoff — APPROVED** [review]

**Comment by thomassedlmayer** [comment]
We should probably also remove the format choice of the osi2read argparser and the respective code in the OSITrace.py on the separated txt format.

**Comment by PhRosenberger** [comment]
> We should probably also remove the format choice of the osi2read argparser and the respective code in the OSITrace.py on the separated txt format.

I just updated the PR accordingly. Could you please check / test it?

**Comment by PhRosenberger** [comment]
@pmai obviously, removing the format command line argument interfers with our current pypeline. Should we change the test of the python scripts in this PR, as well?

**Comment by PhRosenberger** [comment]
> @pmai obviously, removing the format command line argument interfers with our current pypeline. Should we change the test of the python scripts in this PR, as well?

Ok, seems like, I just missed one line, pipeline has passed now.
However, would appreciate some double-check here.

File: doc/architecture/trace_file_formats.adoc [hunk 1]
```diff
@@ -5,18 +5,14 @@ endif::[]
...
 The length is represented by a four-byte, little-endian, unsigned integer.
 The length does not include the integer itself.
 
-*.txt::
-Plain-text trace file.
-Messages are separated by `$$__$$`.
-
 *.txth::
 Human-readable plain-text trace file.
 Messages are separated by newlines.
```
[the remark this answers is not in the document]
Comment (PhRosenberger) [reply doc/architecture/trace_file_formats.adoc:18]: Yes, sounds good to me, as communicating what happened is always agood idea.

**Comment by pmai** [comment]
CCB 2024-02-26: Merge with suggested addition of historical note; thanks for the effort.

**Comment by thomassedlmayer** [comment]
Sorry for the late check. I found another format_type appearance in the file in the osi2read.py.

**Pull request merged**